lib/resources/yum.rb

```diff
@@ -33,7 +32,7 @@
 module Inspec::Resources
   class Yum < Inspec.resource(1)
     name 'yum'
-    desc 'Use the yum InSpec audit resource to test packages in the Yum repository.'
+    desc 'Use the yum InSpec audit resource to the configuration of Yum repositories.'
```

Comment:
This is missing the word "test" and should be:

```desc 'Use the yum InSpec audit resource to test the configuration of Yum repositories.'```

Sorry for being pedantic :)